app/views/service_calendars/master_calendar/otf/_otf_calendar.html.haml

```diff
@@ -23,8 +23,9 @@
       %h3.panel-title
         = t(:calendars)[:otf][:header]
     - unless portal
-      .pull-right
-        = render 'service_calendars/master_calendar/otf/otf_freeze_header_button', service_request: service_request
+      - unless review
```

Comment:
Personally, I find nested conditions to be confusing, especially unlesses. You could do:
`if !portal && !review` or `if !(portal || review)` or (I think) `unless portal || review`
Whatever is a little more readable.